client/src/actions/annotation.ts

```diff
@@ -4,21 +4,20 @@ Action creators for user annotation
 import difference from "lodash.difference";
 import pako from "pako";
 import * as globals from "../globals";
+import { AppDispatch, GetState } from "../reducers";
 import { MatrixFBS, AnnotationsHelpers } from "../util/stateManager";
 
 const { isUserAnnotation } = AnnotationsHelpers;
 
 export const annotationCreateCategoryAction =
   (newCategoryName: string, categoryToDuplicate: string) =>
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  async (dispatch: any, getState: any) => {
+  async (dispatch: AppDispatch, getState: GetState): Promise<void> => {
     /*
-  Add a new user-created category to the obs annotations.
-
-  Arguments:
-    newCategoryName - string name for the category.
-    categoryToDuplicate - obs category to use for initial values, or null.
-  */
+     * Add a new user-created category to the obs annotations.
```

Comment:
If you reformat comments, please convert to tsdoc style:
* comment goes before the function definition, not inside
* it starts with a `/**` not `/*`
* add `@params`
etc.